### PR TITLE
Update UG

### LINKED ISSUE #113: Update UG

### PR DESCRIPTION
**Pull request by ulricolo7**
Fixes #113

### CODE REVIEW

**Review by marraknas — COMMENTED** [review]
LGTM!

**Comment by ulricolo7** [comment]
Fixes #115
Fixes #117

**Review by PeterHW963 — COMMENTED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ FindingbrUdders is a **desktop app for managing contacts and scheduling meetings
 
    * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 r/mUdder m/cs` : Adds a contact named `John Doe` to the Address Book.
 
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+   * `delete 3` : Deletes the 3rd contact shown in the current list and the Meetings associated with it.
```
Comment (XinghuaJulia) [inline docs/UserGuide.md:49]:
```suggestion
   * `delete 3` : Deletes the 3rd Udder shown in the current Udder list and the Meetings associated with it.
```

**Review by XinghuaJulia — COMMENTED** [review]
LGTM! Great job adding on details in the UG :DDDDD

**Pull request merged**